_apidocs/searchgov-results.md

```diff
@@ -66,12 +66,30 @@ Sites indexed via sitemaps or crawling will use the `/i14y` endpoint. Because mo
   | sort\_by (optional)             | Allowed variables are date and relevance. The default sort is relevance. Add `sort_by=date` to sort by date.
   | sitelimit (optional)             | Limits the results to the subdomains and/or subfolders provided. Multiple values can be passed using a space-separated list. Example: `sitelimit=search.gov/about search.gov/get-started`. The values passed in the sitelimit must be in scope of the Domains list in the Search.gov Admin Center for your search site.
 
+  To enable faceted search, use the parameters below. You must set `include_facets=true` for these parameters to be applied. For more context on how we index content into these fields, see our [metadata documentation](https://search.gov/indexing/metadata.html).
+
+  Facet functionality is only available to the **/i14y** endpoint. 
```

Comment:
available to? available at? from? using? depends on the directionality of "available" - we make it available only to the /i14y endpoint, people need to use the /i14y endpoint to offer filters to their searchers.

I suggest we change this to "Faceted search is available only through the **/i14y** endpoint."

Reply:
updated